src/main/java/org/jpeek/Skeleton.java

```diff
@@ -267,6 +268,18 @@ public FieldVisitor visitField(final int access,
                 access, name, desc, signature, value
             );
         }
+        // @todo #114:30min Find and include the name of the
+        //  variable the method is called by, if possible.
+        //  Currently, only name of the method is retrieved.
+        //  Example 1: `Bar.NAME.length();`
+        //  - Here, `length` is retrieved from the param `name`, but the word
+        //  - NAME` is not included in any of the `visitMethodInsn`
+        //  - arguments.
+        //  Example 2: `src.length();`
+        //  - Here, `length` is retrieved from param `name`,
+        //  - but the word `src` is not included in any of
+        //  - the `visitMethodInsn` arguments.
+        //  This is currently implemented in `visitMethodInsn` down below.
```

Comment:
@rok-povsic What do you mean, "this is currently implemented in the method below"? If it is implemented, what is the puzzle for? I think you should rephrase this sentence :)